web_responsive/README.rst

```diff
@@ -52,6 +54,8 @@ this module.
 * Adding ``oe_main_menu_navbar`` ID to the top navigation bar triggers some
   great styles, but also `JavaScript that causes issues on mobile
   <https://github.com/OCA/web/pull/446#issuecomment-254827880>`_
+* The kanban view not is shown if an action window has no kanban mode view
```

Comment:
IMO this isn't worth noting as an issue here. It's inferred that a Kanban won't be shown if there's not a Kanban to show